### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by nzhtbyn** [NREL/floris#773]
We, at the Fluid Mechanics and Turbulence research group at Aarhus University, Denmark, recently had the opportunity to explore the latest version of Floris (v-3.5), and must say that we are impressed with the enhancements and improvements you have made. Your team’s dedication to advancing the field of modeling and control of wind farms is evident in the quality of your work.

We wanted to bring to your attention a particular aspect of the code that we believe could benefit from further refinement. Upon reviewing the “wake_turbulence” packages of the Floris v-3.5, we noticed a discrepancy in how the added turbulence through the formula suggested by [Crespo and Hernández (1996)](https://doi.org/10.1016/0167-6105(95)00033-X) is being handled, where the turbine-induced added turbulence is positively proportional to the ambient turbulence level.

We are specifically referencing the code within the following files:

 `floris/floris/simulation/wake_turbulence/crespo_hernandez.py`

# Problem statement
We recently published a note titled “[A short note on turbulence characteristics in wind-turbine wakes](https://doi.org/10.1016/j.jweia.2023.105504)” which focused on the turbulence characteristics in wind-turbine wakes. The empirical study of [Crespo and Hernández (1996)](https://doi.org/10.1016/0167-6105(95)00033-X), as highlighted in our work, showed that the turbulence added by the wind turbines is negatively proportional to the ambient turbulence level as $I_{+} = 0.73 a^{0.8325} I_0^{-0.0325} \left({x}/d_0\right)^{-0.32}$, where $I_0$, $a$, $x$, and $d_0$ denote the ambient turbulence intensity, axial induction factor of the turbine, the streamwise location, and the rotor diameter, respectively, and $I_+$ corresponds to the added turbulence intensity due to the presence of wind turbines.

We would like to respectfully recommend that you consider making the necessary corrections and refinements to ensure that your code accurately represents the physics involved in turbine wakes.  

# Impact quantificaion
To quantify the impact, we utilize the analytical-empirical Gaussian wake model proposed by [Niayifar and Porte-Agel (2016)](https://doi.org/10.3390/en9090741) as the main framework with the original and the incorrect implementation for the turbine-induced added turbulence. We then apply the wake model to the Horns Rev 1 wind farm to evaluate the turbulence intensity within the farm in the full-wake condition. 

The figure below depicts the turbulence intensity averaged across the rotor upstream of each turbine row, obtained from large-eddy simulations performed by [Wu and Porte-Agel (2015)](https://doi.org/10.1016/j.renene.2014.06.019), the correlation of [Crespo and Hernández (1996)](https://doi.org/10.1016/0167-6105(95)00033-X), and its incorrect implementation (labeled as Eq. (2)). The layout of the HR1 wind farm is also shown in this figure on the right. We can see an absolute difference of 1.94\% between the turbulence intensity of the waked turbines predicted by the correct and the incorrect added-turbulence formulation. The reason behind disagreement and further assessment can be found in the study of [Zehtabiyan-Rezaie and Abkar (2023)](https://doi.org/10.1016/j.jweia.2023.105504).

![287249427-f9fcf64a-c792-4947-82e4-e19896e5d616](https://github.com/NREL/floris/assets/43416436/8b25e2ef-3335-4d58-baf6-e57f21581a7b)

_Figure: N. Zehtabiyan-Rezaie, M. Abkar, “A short note on turbulence characteristics in wind-turbine wakes”, Journal of Wind Engineering and Industrial Aerodynamics, vol. 240, p. 105504, 2023. [10.1016/j.jweia.2023.105504](https://doi.org/10.1016/j.jweia.2023.105504) - Copyright © 2023 by Elsevier._

# References
```
@article{Crespo_Hernandez_1996,
author = {Crespo, A. and Hern{\'{a}}ndez, J.},
doi = {10.1016/0167-6105(95)00033-X},
journal = {Journal of Wind Engineering and Industrial Aerodynamics},
number = {1},
pages = {71--85},
title = {{Turbulence characteristics in wind-turbine wakes}},
volume = {61},
year = {1996}
}

@article{Zehtabiyan_Abkar_2023,
author = {Zehtabiyan-Rezaie, Navid and Abkar, Mahdi},
doi = {https://doi.org/10.1016/j.jweia.2023.105504},
journal = {Journal of Wind Engineering and Industrial Aerodynamics},
pages = {105504},
title = {{A short note on turbulence characteristics in wind-turbine wakes}},
volume = {240},
year = {2023}
}

```

**Comment by misi9170** [comment]
Thank you, @nzhtbyn , for this note, and I'm sorry that it has taken us so long to come to addressing it.

Looking at our implementation in floris/core/wake_turbulence/crespo_hernandez.py, the implementation of the model is in [these lines that use numexpressions](https://github.com/NREL/floris/blob/cc9de0250b6eca1f0da3df164469d11a31ceb634/floris/core/wake_turbulence/crespo_hernandez.py#L80-L85); however, they are equivalent to
```python
 ti = constant * (axial_induction ** ai) * (ambient_TI ** initial) * ((delta_x / rotor_diameter) ** downstream)
```
where, comparing to Equation (1) from [your note](https://www.sciencedirect.com/science/article/pii/S0167610523002076?via%3Dihub), we have:
- `constant` maps to the 0.73 term
- `axial_induction` maps to $a$
- `ai` maps to the 0.8325 term (not the best choice of variable name on our part!)
- `ambient_TI` maps to $I_0$
- `initial` maps to -0.0325 (/0.0325---more on this below)
- `(delta_x / rotor_diameter)` maps to $x/d_0$
- and `downstream` maps to -0.32

To address your point, I at first thought we can simply change the default value for `initial` to -0.0325, which, as you point out, was what has intended by [Crespo and Hernández](https://www.sciencedirect.com/science/article/pii/016761059500033X?via%3Dihub). However, I see that the default value we have for `initial` is _not_ 0.0325, as I was expecting, [but instead 0.1](https://github.com/NREL/floris/blob/cc9de0250b6eca1f0da3df164469d11a31ceb634/floris/core/wake_turbulence/crespo_hernandez.py#L49). This value of 0.1 also appears in various example input files, for example [the main GCH input file](https://github.com/NREL/floris/blob/cc9de0250b6eca1f0da3df164469d11a31ceb634/examples/inputs/gch.yaml#L240).

I've gone back through the FLORIS history, and it appears that we have had this value for `initial` for a long time: it appears that the default values have changed several times (e.g. in #79) and were possibly first introduced in #72; but as far as I can tell, we have not ever had the defaults at 0.0325 (or -0.0325). I am tagging @bayc and @paulf81 , who may know more about why the default values are as they are.

**Comment by nzhtbyn** [comment]
Hi @misi9170, and Happy New Year!

Thank you for your response and for the historical context on the default parameter values. It is understandable that these defaults may have evolved over time through optimization against available datasets. My objective in raising this issue is to discuss how we might ensure the underlying functional form of the wake-added turbulence model remains consistently aligned with established physical principles.

A consistent observation in the literature is that the turbine-added turbulence component becomes smaller as ambient turbulence increases. This occurs because higher inflow turbulence enhances background mixing, accelerating the breakdown of turbine-induced structures. The original [Crespo and Hernández](https://doi.org/10.1016/0167-6105(95)00033-X) formulation reflects this effect through a negative exponent for $I_0$. Some subsequent citations and implementations in the literature appear to have used a positive exponent for $I_0$ within _this very model_. It is likely due to the low quality of early document scans, where the negative sign was easily overlooked.

Several later works attempted to refine the Crespo and Hernández model by incorporating broader datasets, often optimizing coefficients and exponents to minimize numerical error. While such approaches can improve agreement with specific cases, it is important that the model remains physically grounded. In particular, the functional form should ensure that increases or reductions in individual parameters produce physically consistent responses rather than relying on compensating empirical tuning. **Interpreted physically, a positive exponent for $I_0$ implies that increasing inflow turbulence amplifies the turbine’s turbulence contribution, which runs counter to the well-established physical understanding.**

As engineering wake models continue to evolve, maintaining a physically grounded functional form is essential for robust predictions. As highlighted in [our recent work](https://doi.org/10.1016/j.weer.2024.100007), the treatment of wake-added turbulence can propagate into meaningful differences at both intra-farm and inter-farm scales.

Thank you again for your openness in discussing this topic and for the ongoing development of FLORIS, which continues to have a significant impact on the wind-energy community.

**Comment by misi9170** [comment]
Thanks @nzhtbyn , understood!

To avoid changing the FLORIS model outputs at this time, but to still capture your concern, I've added some comments in various places in #1171, which will soon be merged into the develop and eventually into the main branch of FLORIS.

In particular, I've added the following text [to the docstring](https://github.com/NREL/floris/blob/a1d836b33f6188694ca3fa16b62de8a4461d48e1/floris/core/wake_turbulence/crespo_hernandez.py#L26-L40) for the `CrespoHernandez` class:
```
 Note: The values for default parameters provided here differ from those in
    :cite:`cht-crespo1996turbulence. Following their recommendations, the
    default parameters would instead be:
        - initial: -0.0325*
        - constant: 0.73
        - ai: 0.8325
        - downstream: -0.32
    * The "initial" parameter is given as -0.0325 in :cite:`cht-crespo1996turbulence`,
    but the negative exponent is not clear in the scans of the paper found on the internet,
    and several subsequent paper cite the exponent as positive (0.0325). This discrepancy
    is noted in :cite:`zehtabiyan_rezaie_CH_2023`. Moreover, :cite:`zehtabiyan_rezaie_CH_2023`
    argues that positive values for this exponent are not representative of the physical
    phenomena occurring. For more details, see https://github.com/NREL/floris/issues/773.
    None-the-less, the default value here is set to 0.1 for consistency with previous
    FLORIS versions. The default value may be updated in a future release.
```

Once #1171 is merged into develop, this note will also appear in the API documentation on [this page](https://nrel.github.io/floris/_autosummary/floris.core.wake_turbulence.crespo_hernandez.html#module-floris.core.wake_turbulence.crespo_hernandez).

Finally, I've also updated the [description of the Crespo-Hernández model in our documentation](https://nrel.github.io/floris/wake_models.html#crespo-hernandez) to say:
___
### Crespo-Hernandez

CrespoHernandez is a wake-turbulence model that is used to compute additional variability introduced
to the flow field by operation of a wind turbine. Implementation of the model follows the original
formulation and limitations outlined in {cite:t}`crespo1996turbulence`.

The default parameter values used in FLORIS for CrespoHernandez differ from those reported in {cite:t}`crespo1996turbulence` following subsequent calibration. However, {cite:t}`zehtabiyan_rezaie_CH_2023` argue that the sign of certain parameters are not physically consistent (and also misreported in subsequent literature). See the `CrespoHernandez` class docstring for more details.
___

This will also appear in the documentation once #1171 is merged.

In both of these locations, the citation links (`:cite:`zehtabiyan_rezaie_CH_2023`` and  `{cite:t}`zehtabiyan_rezaie_CH_2023` `) will lead to references for your paper so that readers can find details about the sign mismatch.

**Comment by nzhtbyn** [comment]
Hi @misi9170, and thanks for looking into this and adding the notes.
